src/dotnet/Host/appsettings.Development.json

```diff
@@ -30,11 +30,8 @@
   },
   "UsersSettings": {
     "Db": "",
-    "Redis": ""
-
-
-
-
+    "Redis": "",
+    "NewUserDefaultStatus": "Active"
```

Comment:
Why not `NewUserStatus`?

Reply:
Or e.g. `DefaultUserStatus`, if there is some way to override it. I.e. one of these two words seems excessive...

Reply:
done